File: Website/Orientawjih V.0/fct/classes/Choix.php
<?php
class Choix{
    public $id_ad=-1,$id_et=-1,$id,$diplomes,$lieus;
    public $name,$desc,$date;
    public $horizons,$fillieres,$conditions;
    function __construct($id, $name, $desc) {
        $this->id = $id;
        $this->name = $name;
        $this->desc = $desc;
    }
}
class Filliere{
    public $id;
    public $name;
    function __construct($id,  $name) {
        $this->id = $id;
        $this->name = $name;
    }
}
class Diplome{
    public $id;
    public $nom;
    public $duree;

    function __construct($id,  $nom, $duree) {
        $this->id = $id;
        $this->nom = $nom;
        $this->duree = $duree;
    }
}
class Horizon{
    public $id;
    public $name;
    //don't add diplmoes here cuz we xont have only one horizon
    function __construct($id,$name) {
        $this->id = $id;
        $this->name = $name;
    }
}
class Condition{
    public $age_d,$age_f;
    public $Note_seuile;
    public $nbr_places,$id;
    function __construct($id,$age_d,$age_f, $note_seuile,$nbr_places) {
        $this->nbr_places=$nbr_places;
        $this->id=$id;
        $this->age_d = $age_d;
        $this->age_f=$age_f;
        $this->Note_seuile = $note_seuile;
    }
}
class Lieu{
    public $id;
    public $name;
    public $Long,$Latt;
    function __construct($id, $name, $aLong, $latt) {
        $this->id = $id;
        $this->name = $name;
        $this->Long = $aLong;
        $this->Latt = $latt;
    }
}
 ?>

File: Android/app/src/main/java/ma/me/amirez/orientawjih/Choix.java
package ma.me.amirez.orientawjih;

import java.util.ArrayList;

/**
 * Created by HP on 26/04/2017.
 */
public class Choix {
    int id;
    String name,desc;
    ArrayList<Condition>conditions;
    ArrayList<Fillere> filleres;
    ArrayList<Horizon>horizons;
    ArrayList<Diplome>diplomes;
    ArrayList<Lieu>lieus;

    public Choix(int id, String name, String desc, ArrayList<Condition> conditions, ArrayList<Fillere> filleres, ArrayList<Horizon> horizons, ArrayList<Diplome> diplomes, ArrayList<Lieu> lieus) {
        this.id = id;
        this.name = name;
        this.desc = desc;
        this.conditions = conditions;
        this.filleres = filleres;
        this.horizons = horizons;
        this.diplomes = diplomes;
        this.lieus = lieus;
    }

    public Choix(int id, String name, String desc) {
        this.id = id;
        this.name = name;
        this.desc = desc;
    }
}
class Fillere{
    int id;
    String name;

    public Fillere(int id, String name) {
        this.id = id;
        this.name = name;
    }
}
class Diplome{
    int id;
    String nom;
    int duree;

    public Diplome(int id, String nom, int duree) {
        this.id = id;
        this.nom = nom;
        this.duree = duree;
    }
}
class Horizon{
    int id;
    String name;
    //don't add diplmoes here cuz we xont have only one horizon
    public Horizon(int id, String name) {
        this.id = id;
        this.name = name;
    }
}
class Condition{
    int age;
    //montion declaree comme enumeration
    int Note_seuile;
    public Condition(int age, int note_seuile, ArrayList<Fillere> filleres) {
        this.age = age;
        Note_seuile = note_seuile;
    }
}
class Lieu{
    int id;
    String name;
    int Long,Latt;
    public Lieu(int id, String name, int aLong, int latt) {
        this.id = id;
        this.name = name;
        Long = aLong;
        Latt = latt;
    }
}
